traitsui/qt4/helper.py

```diff
@@ -173,12 +173,20 @@ def __init__(self, icon, slot):
 
         # Configure the button.
         self.setIcon(ico)
-        self.setMaximumSize(ico_sz, ico_sz)
         self.setFlat(True)
         self.setFocusPolicy(QtCore.Qt.NoFocus)
 
         self.clicked.connect(slot)
 
+    def sizeHint(self):
+        # We want the button to have a size similar to the icon.
+        # Using the size computed for a tool button gives a desirable size.
```

Comment:
Could we add a docstring for this method?  The block comment covers the majority of the important information but having a docstring would be good for consistency

Reply:
Yep. Done!